katib/katib-controller/v3/kustomization.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  - ../base/katib-configmap.yaml
```

Comment:
So per comment: https://github.com/kubeflow/manifests/pull/1124#issuecomment-618634808

I think this should be something like

resources:
- ../katib-system
- ../katib-db-manager
- ../katib-mysql

The resources listed here would then move into the respective packages.

patchesStrategicMerge
- <patch to add MySQL env variables to db manger>

Reply:
Yes I agree, we can use patch here.
What do you think @discordianfish, can you make these changes in https://github.com/kubeflow/manifests/pull/986 PR?

/cc @johnugeorge @gaocegege

Reply:
I think in #986 @discordianfish said he didn't have time to make any additional changes. So maybe @andreyvelich you could just make the changes in this PR? Or a follow on PR?

Reply:
I was waiting for you to settle on one way to do it. But now it's probably better to merge this here first. I can look into a new PR to change this.